docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+* link:https://issues.redhat.com/browse/MTA-823[MTA-823] - Report custom metrics on MTA usage back via OpenShift Telemetry
+* link:https://issues.redhat.com/browse/MTA-402[MTA-402] - Report custom metrics on MTA usage back via OpenShift Telemetry
+* link:https://issues.redhat.com/browse/MTA-402[MTA-402] - Downstream efforts for MTA 6.2.0
+* link:https://issues.redhat.com/browse/MTA-894[MTA-894] - Failed analysis counted twice
+* link:https://issues.redhat.com/browse/MTA-872[MTA-872] - After an error, fails to export same applications as tastks, returns error showing sub-tasks
+* link:https://issues.redhat.com/browse/MTA-973[MTA-973] - Success alert missing when creating a new Jira instance
+* link:https://issues.redhat.com/browse/MTA-810[MTA-810] - Fails to run analysis in provate GitHub repo
+* link:https://issues.redhat.com/browse/MTA-984[MTA-984] - Dependencies: error retreiving data with `Unable to Connect`
```

Comment:
* link:https://issues.redhat.com/browse/MTA-984[MTA-984] - Dependencies: error **retrieving** data with `Unable to Connect`